app/src/views/RoleCategoriesView.vue

```diff
@@ -65,11 +65,10 @@ const roleCategory = ref({
 })
 
 const {
-  error: roleCategoryError, 
-  data: _roleCategories, 
+  error: roleCategoryError,
+  data: _roleCategories,
```

Comment:
Why this syntaxes ?

Reply:
One is for data for displaying RoleCategoryCards with role categories from the database and the other is for creating a new role category with AddRoleCategoryForm